inst/shiny-examples/SCD-effect-sizes/server.R

```diff
@@ -435,7 +441,28 @@ shinyServer(function(input, output, session) {
     }
   }) 
   
-  output$datview2 <- renderTable(datClean())
+  datClean2 <- reactive({
+    dat <- datClean()
+    
+    if (input$calcPhasePair) {
+      grouping_vars <- input$b_clusters
+      session_var <- input$session_number
+      phase_var <- input$b_phase
+      
+      dat <- 
+        dat %>% 
+        dplyr::group_by(!!!rlang::syms(grouping_vars)) %>% 
+        dplyr::arrange(!!!rlang::syms(grouping_vars), !!rlang::sym(session_var)) %>% 

```

Comment:
Skip the `arrange()` step and use the `session` argument of `calc_phase_pairs()`.